Designer/ViewModel/ViewDesignViewModel.cs

```diff
@@ -44,6 +45,45 @@ public class ViewDesignViewModel : INotifyPropertyChanged {
         private ProductPlacement _draggingPlacement;
         public Polygon RoomPoly { get; set; }
         public bool AllowDrop = false;
+        public int DistanceScore { 
+            get
+            {
+                int increment = 0;
+                if (ProductPlacements == null) return 0;
+
+                List<ProductPlacement> placements = ProductPlacements.ToList();
+
+                //Loopt door alle paren van producten zonder overbodige stappen zoals p1 -> p1 en p1 -> p2, p2 -> p1
+                for (int i = 0; i < placements.Count; i++)
+                {
+                    bool noDistance = false;
+                    ProductPlacement placement1 = placements[i];
+                    for (int j = 0; j < placements.Count; j++)
```

Comment:
Als je hier j op i start is de volgende if niet nodig

Reply:
> Als je hier j op i start is de volgende if niet nodig

Heb dit geprobeerd maar werkte niet, daarnaast ook in de conditie proberen te verwerken maar werkte ook niet.